mula/scheduler/schedulers/scheduler.py

```diff
@@ -104,12 +102,85 @@ def post_push(self, p_item: models.PrioritizedItem) -> None:
             modified_at=datetime.now(timezone.utc),
         )
 
+        if p_item.hash is None:
+            self.logger.warning(
+                "Task %s has no hash, not creating job [task_id=%s, queue_id=%s]",
+                p_item.data.get("id"),
+                p_item.data.get("id"),
+                self.queue.pq_id,
+            )
+            return
+
         task_db = self.ctx.task_store.get_task_by_id(str(p_item.id))
-        if task_db is not None:
+        if task_db is None:
+            task_db = self.ctx.task_store.create_task(task)
+        else:
             self.ctx.task_store.update_task(task)
+
+        if task_db is None:
+            self.logger.warning(
+                "Task %s could not be created, not creating job [task_id=%s, queue_id=%s]",
+                p_item.data.get("id"),
+                p_item.data.get("id"),
+                self.queue.pq_id,
+            )
+            return
+
+        # When there is already a job with the same hash, we update the job
```

Comment:
Check this for normalizer scheduler. Do we want to create jobs for the normalizer? Probably needs to be configurable per scheduler for rescheduling functionality.